bundles/org.openhab.binding.renault/src/main/java/org/openhab/binding/renault/internal/handler/RenaultHandler.java

```diff
@@ -196,7 +202,16 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                         return;
                     }
                 }
+                break;
             default:
+                if (command instanceof RefreshType) {
```

Comment:
You might want to do this before checking the channel, since the job will probably refresh values for all channels? With the current implementation, it seems that a request to refresh one of the channels above will not work?

Reply:
Good point.  Yes a REFRESH will poll the car and update all channels.  This is because the poll follows the phyton logic and goes through all services.  (If I have time to debug the API request limit counters I might be able to improve this with a later enhancement)

I removed 2 breaks for channels that REFRESH by contacting the car.  The other channel "CHANNEL_HVAC_TARGET_TEMPERATURE" has its own REFRESH logic and does not contact the car so it has a break.  I think this is better and correct.

Reply:
It doesn't look correct. You will now update both **CHANNEL_HVAC_STATUS** and **CHANNEL_CHARGING_MODE** with a value intended for **CHANNEL_HVAC_STATUS** when receiving a command which is not `RefreshType`. In the beginning of the method you might want to do something like this instead:

```java
if (command instanceof RefreshType) {
    switch (channelUID.getId()) {
        case RenaultBindingConstants.CHANNEL_HVAC_STATUS:
        case RenaultBindingConstants.CHANNEL_CHARGING_MODE:
            // Reschedule job for supported channels
    }
    return;
}
```

Reply:
??? Sorry I am confused... What I think I want is that an action will be performed if the channel id and command specify it.  CHANNEL_HVAC_TARGET_TEMPERATURE and REFRESH have their own logic block and a break. All other channels trigger a refresh of all channels if the command is a REFRESH?

At the moment a poll updates all channels.

Reply:
Context: https://github.com/openhab/openhab-addons/pull/13970#discussion_r1052653841

So my understanding is that you would like the possibility to refresh on demand, for example from a rule. This is what `RefreshType` is for. However, this should not be mixed with other commands. You will either send a command to a channel in order to update its state, like:

```java
Renault_ChargingMode.SendCommand("ALWAYS_CHARGING");
```

or you will send a refresh command in order to request a state update from the thing:

```java
sendCommand(Renault_ChargingMode, "REFRESH")
```

What is a little confusing here is that updating CHANNEL_HVAC_STATUS will also reschedule the job. I'm not sure what the reason for that is, but maybe this is some kind of work-around to reflect the new state.

Reply:
@jlaur  OK now I have a per channel handler with a break, and a default for the other channels.  This is nice and simple and I think will avoid the issue you describe and future confusion.

Reply:
The CHANNEL_HVAC_STATUS reschedules the polling with a delay of a few seconds (config.updateDelay).  This gives the Renault servers time to contact the car and send the command to it.  They then update their HVAC status for the car.  Then we can query it and change our status from "PENDING" to "ON/OFF"